src/dotnet/CarbonAware.Aggregators/src/Configuration/ServiceCollectionExtensions.cs

```diff
@@ -12,4 +13,10 @@ public static void AddCarbonAwareEmissionServices(this IServiceCollection servic
         services.AddDataSourceService(DataSourceType.JSON);
         services.TryAddSingleton<ICarbonAwareAggregator, CarbonAwareAggregator>();
     }
+
+     public static void AddCarbonAwareSciScoreServices(this IServiceCollection services)
+    {
+        services.AddDataSourceService(DataSourceType.WattTime);
```

Comment:
Will this conflict with the environment variable change that makes the data source selectable?

Reply:
Done!